src/IO/H5/Python/CombineH5Dat.py

```diff
@@ -8,37 +8,39 @@
 import click
 import h5py
 
+from spectre.Visualization.ReadH5 import available_subfiles
 
-def combine_h5_dat_segments(output, input_files):
+
+def combine_h5_dat(**kwargs):
```

Comment:
Take the arguments explicitly here. Also document them please (see other py files for the syntax).